trie/database.go

```diff
@@ -85,7 +85,7 @@ func (db *Database) LoadGenesisData() (*genesis.GenesisData, error) {
 	}
 
 	data, err := scale.Decode(enc, &genesis.GenesisData{
-		Bootnodes: [][]byte{{}},
+		Bootnodes: [][]byte{{}}, // TODO: read slice length from SCALE encoding
```

Comment:
This was the root cause of the gossamer command not being able to support multiple bootnodes listed in the genesis file. We were attempting to load bootnodes from the db where we are using SCALE encoding but with a defined number of bootnodes in the interface provided here, otherwise encoding panics on `reflect.value.Index()` using this interface.

Reply:
addressed in #539